services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/cache/HostResourceIdCache.java

```diff
@@ -0,0 +1,43 @@
+package com.futurewei.alcor.netwconfigmanager.cache;
+
+import com.futurewei.alcor.common.db.CacheFactory;
+import com.futurewei.alcor.common.db.ICache;
+import com.futurewei.alcor.common.stats.DurationStatistics;
+import com.futurewei.alcor.netwconfigmanager.entity.ResourceMeta;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@ComponentScan(value = "com.futurewei.alcor.common.db")
+public class HostResourceIdCache {
+    // Map <HostId, List<ResoruceIDType>>
+    private ICache<String, ResourceMeta> hostResourceMetas;
```

Comment:
@xieus Is HostResourceIdsCache better?

Reply:
I think HostResourceIdCache is fine becuase cache of something is more common than cahce of somethings.